components/contact/ContactForm2.vue

```diff
@@ -0,0 +1,813 @@
+              class="formSelect"
+              @change="onSector($event)"
+            >
+              <option disabled value="">
+                {{ $t('contact.selSector') }}
+              </option>
+              <option value="notProfit">
+                {{ $t('contact.notProfit') }}
+              </option>
+              <option value="forProfit">
+                {{ $t('contact.forProfit') }}
+              </option>
+              <option value="provGovCoorp">
+                {{ $t('contact.provGovCoorp') }}
+              </option>
+              <option value="fedGovDept">
+                {{ $t('contact.fedGovDept') }}
+              </option>
+            </select>
+          </div>
+        </div>
+
+        <div class="flex mb-4">
+          <div class="w-5/12 margins">
+            <label class="formLabel" for="contributionRefNo">
+              {{ $t('contact.contrib') }}
+            </label>
+            <input
+              id="contributionRefNo"
+              v-model="contactInfo.contributionRefNo"
+              class="formInput"
+              type="text"
+              :placeholder="$t('contact.contrib')"
+            />
+          </div>
+          <div class="w-5/12 margins">
+            <label class="formLabel" for="serviceContrNo">
+              {{ $t('contact.service') }}
+            </label>
+            <input
+              id="serviceContrNo"
+              v-model="contactInfo.serviceContrNo"
+              class="formInput"
+              type="text"
+              :placeholder="$t('contact.service')"
+            />
+          </div>
+        </div>
+
+        <div class="flex mb-4">
+          <div class="w-5/12 margins">
+            <label class="formLabel" for="onStandingOffer">
+              {{ $t('contact.standing') }}
+            </label>
+            <select
+              id="onStandingOffer"
+              v-model="contactInfo.onStandingOffer"
+              class="formSelect"
+              @change="onStanding($event)"
+            >
+              <option value="false">
+                {{ $t('contact.no') }}
+              </option>
+              <option value="true">
+                {{ $t('contact.yes') }}
+              </option>
+            </select>
+          </div>
+        </div>
+
+        <div
+          v-if="message.message != null"
+          class="messageBox text-lg"
+          :class="[message.type == 'error' ? ' error' : ' ']"
+        >
+          <span>
+            {{ message.message }}
+          </span>
+        </div>
+
+        <div class="flex justify-start mb-4">
+          <div class="w-3/12 margins">
+            <AppButton custom_style="btn-cancel" data_cypress="formButton">
+              {{ $t('contact.cancel') }}
+            </AppButton>
+          </div>
+          <div class="w-3/12 margins">
+            <AppButton custom_style="btn-extra" data_cypress="formButton">
+              {{ $t('contact.save') }}
+            </AppButton>
```

Comment:
These buttons have identical behaviour still. Cancel should not save the new info or notify the user that the contact has been saved.